pkg/reconciler/route/resources/ingress.go

```diff
@@ -194,9 +194,18 @@ func makeIngressSpec(
 		}
 	}
 
+	var httpOption netv1alpha1.HTTPOption
+	if config.FromContext(ctx).Network.HTTPProtocol == network.HTTPEnabled {
+		httpOption = netv1alpha1.HTTPOptionEnabled
+	}
+	if config.FromContext(ctx).Network.HTTPProtocol == network.HTTPRedirected {
+		httpOption = netv1alpha1.HTTPOptionRedirected
+	}
```

Comment:
If this defaults to enabled, which I think it does, then I believe it will break downgrades because resources will have a field that the earlier version thinks they should not.

If this can wait until after the cut, so that the field is present in the N-1 release, then I think this is fine.

Reply:
Let's wait for the release cut.